app/models/label/LabelTable.scala

```diff
@@ -638,6 +655,283 @@ object LabelTable {
     selectedLabels
   }
 
+  /**
+   * Retrieves n labels of specified type, severities, and tags.
+   *
+   * @param labelTypeId Label type specifying what type of labels to grab.
+   * @param n Number of labels to grab.
+   * @param loadedLabels Set of labelIds already grabbed as to not grab them again.
```

Comment:
These two parameter names in the comments don't match with the actual parameter names.

Reply:
Fixed